looper/__init__.py

```diff
@@ -18,7 +18,7 @@
 from .const import *
 
 from .conductor import write_sample_yaml_cwl, write_sample_yaml, \
-    write_sample_yaml_prj
+    write_sample_yaml_prj, write_namespaces
```

Comment:
follow the plugin naming convention we've established. So that would be: "write_submission_yaml".